src/commonMain/kotlin/macroBenchmarks/Coroutines.kt

```diff
@@ -0,0 +1,37 @@
+package macroBenchmarks
+
+import macroBenchmarks.coroutines.AbstractGenerator
+import macroBenchmarks.coroutines.ClosedRangeGenerator
+import macroBenchmarks.coroutines.CoroutineStub
+import macroBenchmarks.coroutines.RecursiveFibonacciGenerator
+
+sealed class Coroutines: MacroBenchmark() {
```

Comment:
I'm wondering if there is any reasoning behind this case. Why did you choose these cases? 
It would be nice to add a comment about that if there any.